packages/x-components/src/x-modules/extra-params/components/snippet-config-extra-params.vue

```diff
@@ -59,6 +66,9 @@
       ...snippetExtraParams
     }: SnippetConfig): void {
       forEach(snippetExtraParams, (name, value) => {
+        if (this.notAllowedExtraParams.includes(name)) {
```

Comment:
I was trying to find a "clean" solution because I dont like to make an empty return but i dont like the other solution either:

`  if (!this.notAllowedExtraParams.includes(name)) {
          this.$set(this.extraParams, name, value);
    }`
    
 I dont know if you have more suggestions hahah if not, i will give my approve

Reply:
I think either way is correct. The one he choose is more _backendish_

Reply:
Yes, I prefer to use always positive statements, so lets go ahead with this solution